ciao/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
This file shouldn't exist.  I realise it may have been added by cobra but we should remove it as we already have a license one level up.